addons/web/static/src/libs/fontawesome/css/font-awesome.css

```diff
@@ -1,13 +1,13 @@
 /*!
- *  Font Awesome 4.7.0 by @davegandy - http://fontawesome.io - @fontawesome
+ *  Based on Font Awesome 4.7.0 by @davegandy - http://fontawesome.io - @fontawesome
  *  License - http://fontawesome.io/license (Font: SIL OFL 1.1, CSS: MIT License)
  */
 /* FONT PATH
  * -------------------------- */
 @font-face {
   font-family: 'FontAwesome';
-  src: url('../fonts/fontawesome-webfont.eot?v=4.7.0');
-  src: url('../fonts/fontawesome-webfont.eot?#iefix&v=4.7.0') format('embedded-opentype'), url('../fonts/fontawesome-webfont.woff2?v=4.7.0') format('woff2'), url('../fonts/fontawesome-webfont.woff?v=4.7.0') format('woff'), url('../fonts/fontawesome-webfont.ttf?v=4.7.0') format('truetype'), url('../fonts/fontawesome-webfont.svg?v=4.7.0#fontawesomeregular') format('svg');
+  src: url('../fonts/fontawesome-webfont.woff?v=4.7.0') format('woff');
```

Comment:
To respect the priority, shouldn't we add woff2 at first?

Reply:
```css
src: url('../fonts/fontawesome-webfont.woff?v=4.7.0') format('woff');
src: url('../fonts/fontawesome-webfont.woff2?v=4.7.0') format('woff2');
```
=> woff 2 

```css
src: url('../fonts/fontawesome-webfont.woff2?v=4.7.0') format('woff2');
src: url('../fonts/fontawesome-webfont.woff?v=4.7.0') format('woff');
```
=> woff 1

```css
src: url('../fonts/fontawesome-webfont.woff?v=4.7.0') format('woff'), url('../fonts/fontawesome-webfont.woff2?v=4.7.0') format('woff2');
```
=> woff 1

```css
src: url('../fonts/fontawesome-webfont.woff2?v=4.7.0') format('woff2'), url('../fonts/fontawesome-webfont.woff?v=4.7.0') format('woff');
```
=> woff 2 :ok_hand: <= Best solution

tested in Chrome 98 et FF 97